public/index.html

```diff
@@ -2,42 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="" href="" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
`<link rel="preconnect">` lets your browser know that your page intends to establish a connection to another origin before an HTTP request is actually sent to the server saving you some time but as the fonts are external requests made by the googleapis stylesheet it wouldn't make too much of a difference in load time. As you'd be preconnecting to the stylesheet not the actual fonts used. 

For that performance boost you could also preconnect to the external fonts that the Google fonts stylesheet will download e.g.
```suggestion
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preload" href="https://fonts.gstatic.com/s/lato/v20/S6u8w4BMUTPHh30AXC-qNiXg7Q.woff2" as="font" crossorigin>
```